src/test/getAssetFromKV.ts

```diff
@@ -17,6 +17,21 @@ test('getAssetFromKV return correct val from KV and default caching', async t =>
     t.fail('Response was undefined')
   }
 })
+test('getAssetFromKV evaluated the file matching the extensionless path first /client/ -> client', async t => {
```

Comment:
shouldn't this map to `client/index.html`, consistent with https://en.wikipedia.org/wiki/Clean_URL?

Reply:
The purpose of this PR is to not exclusively match html files and have raw files be an option (e.g. upload and serve a `.env` )

Reply:
to back this one up: not all major static site generators adhere to that standard